tests/ui/conftest.py

```diff
@@ -0,0 +1,195 @@
+import logging
```

Comment:
not all UI tests are under /tests/ui/ 
Maybe better to place these fixtures directly in tests/conftest.py

Reply:
I agree. Will move. To tell you the true I think that having a UI directory is not a good idea. I think that even it's ui tests it should go to the regulars directory that we have now in the context of the test.
But do I need tier1 again??? :(

Reply:
I believe deployment is sufficient for validation